.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
```

Comment:
should we have java 6 here, at least until some Mac OS person fixes our mac situation?